packages/vats/test/upgrading/device-bridge.js

```diff
@@ -0,0 +1,38 @@
+import { BridgeId } from '@agoric/internal';
+import { Far } from '@endo/far';
+
+export const buildRootDeviceNode = tools => {
+  let inboundHandler;
+  const {
+    endowments: { t },
+  } = tools;
+  const expectedStorageValues = ['abc', 'def'];
```

Comment:
Can you move this into packages/vats/test/upgrading/test-upgrade-vats.js so test details are not split across files?